src/installing-guacamole.md

```diff
@@ -91,11 +91,19 @@ are strictly required *in all cases* - Guacamole cannot be built without them.
   :::{list-table}
   :stub-columns: 1
   * - Debian / Ubuntu package
-    - `libpng12-dev`
+    - `libpng-dev`
   * - Fedora / CentOS / RHEL package
     - `libpng-devel`
   :::
 
+  In some previous versions of Debian such as Debian 8 / Ubuntu 16.04, you need
+  the `libpng12-dev` package.
+
+  :::{list-table}
+  :stub-columns: 1
+  * - Debian / Ubuntu package
+    - `libpng12-dev`
```

Comment:
The terminating `:::` is missing here. For example, see just above:

https://github.com/apache/guacamole-manual/blob/92940e75490dde8bfef3a9f0710553eea1954030/src/installing-guacamole.md?plain=1#L91-L97

Reply:
I see. Thank you for sparing your precious time.